README.md

```diff
@@ -165,43 +165,56 @@ Three keys to achieving good recall are:
 2. Choose an appropriate number of lists - a good place to start is `rows / 1000` for up to 1M rows and `sqrt(rows)` for over 1M rows
 3. When querying, specify an appropriate number of [probes](#query-options) (higher is better for recall, lower is better for speed) - a good place to start is `sqrt(lists)`
 
-Add an index for each distance function you want to use.
+The IVF index supports both `flat` and `8-bit scalar quantized` storage. The `8-bit quantized` vectors map each vector from `float` space to `int8` space, resulting in up to ~2X faster scoring and ~4X lower storage cost with nearly no recall loss.
+
+Add an index for each distance function you want to use. Specify `ivfflat` to use the `flat` storage, or `ivf` with `quantizer = 'SQ8'` option to use the `8-bit quantized` IVF index.
 
 L2 distance
 
 ```sql
+CREATE INDEX ON items USING ivf (embedding vector_l2_ops) WITH (lists = 100, quantizer = 'SQ8');
```

Comment:
Why is this implemented as a new index access method instead of as an option on the existing ivfflat code? Having it as an existing option would make it simpler for users to manage their indexes.

Reply:
We discussed this with @ankane. Our understanding is that `flat` means no encoding - here are some examples from [Milvus](https://milvus.io/docs/v2.0.x/index.md) and [Faiss](https://github.com/facebookresearch/faiss/wiki/Faiss-indexes#:~:text=IndexHNSW%20supports%20the%20following%20Flat%20indexes%3A%20IndexHNSWFlat%20(no%20encoding)%2C%20IndexHNSWSQ%20(scalar%20quantizer)%2C%20IndexHNSWPQ%20(product%20quantizer)%2C%20IndexHNSW2Level%20(two%2Dlevel%20encoding).) and it would be good to be consistent.

Agree that we should make it simpler for users to use. We think `ivf` with an `quantizer` option would provide more flexibility - we could also support `ivf WITH (quantizer = 'flat')` if needed.

Reply:
OR as another examples

```sql
CREATE INDEX ON items USING ivf (lists=100); -- defaults to `quantizer=flat` or `quantizer=NULL` or however one wants to represent "flat"
CREATE INDEX ON items USING ivf (lists=100, quantizer='SQ8'); 
```

That said, given `ivfflat` index AM is out there, we do need to be careful about introducing new access methods. Effectively we need to treat `ivfflat` as if it's not going away. Maybe `ivf` becomes the preferred choice when creating an IVF index and the default is to leverage the `ivfflat` infrastructure.

Reply:
>> Maybe ivf becomes the preferred choice when creating an IVF index and the default is to leverage the ivfflat infrastructure.

We discussed the same proposal with @ankane as well. I can update the PR to support `quantizer='flat'`, aliasing to `ivfflat`. +1 on using `ivf` as the preferred choice.